src/main/java/com/github/devotedmc/hiddenore/Config.java

```diff
@@ -47,24 +51,51 @@ private Config() {
 		ignoreSilktouch = false;
 		defaultPrefix = "You found hidden ore!";
 		transformAttemptMultiplier = 3;
+		configurations = new HashMap<String, FileConfiguration>();
 	}
 
 	public static void loadConfig() {
 		try {
-			file = HiddenOre.getPlugin().getConfig();
-			doLoad();
+			mainConfig = HiddenOre.getPlugin().getConfig();
+			doLoadMainConfig(mainConfig);
+			for (World world : HiddenOre.getPlugin().getServer().getWorlds()) {
+				if(world == null) {
```

Comment:
So, (world == null) -- this should never be. Did you mean to iterate over all worlds in the main Config? Or is this leftover from an earlier iteration?

I also ask because in the very next line you have `world.getName()` which would immediately NullPointerException if indeed `world == null`...